addOns/commonlib/src/main/java/org/zaproxy/addon/commonlib/CommonAlertTag.java

```diff
@@ -80,7 +80,300 @@
             "https://owasp.org/www-project-top-ten/2017/A9_2017-Using_Components_with_Known_Vulnerabilities.html"),
     OWASP_2017_A10_LOGGING_FAIL(
             "OWASP_2017_A10",
-            "https://owasp.org/www-project-top-ten/2017/A10_2017-Insufficient_Logging%2526Monitoring.html");
+            "https://owasp.org/www-project-top-ten/2017/A10_2017-Insufficient_Logging%2526Monitoring.html"),
+
+    // OWASP WSTG v4.2
+    WSTG_V42_INFO_01(
```

Comment:
I'd suggest following the style of the Top 10 enums, ie including part of the name in the enum name, like `OWASP_2021_A01_BROKEN_AC`.

My concern was that when reviewing scan rules its not always obvious is someone used `OWASP_2021_A01`instead of `OWASP_2021_A02` - I think thats even more of a danger with the WSTG ones - I'd have to refer to the list everytime :/

Reply:
Yup, that makes sense. I'll do another pass and see what I can add to make things more obvious from a coding PoV.

Reply:
Ok, I've tweaked all the ENUM names.